### PR TITLE
Add KmPlot

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://bugs.kde.org/show_bug.cgi?id=453851
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* https://invent.kde.org/education/kmplot/-/merge_requests/10

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
cc @flathub/kde

**Comment by hfiguiere** [comment]
bot, build org.kde.kmplot

File: org.kde.kmplot.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        "--socket=wayland",
+        "--device=dri"
+    ],
+    "modules": [
+        {
+            "name": "kmplot",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
```
Comment (travier) [inline org.kde.kmplot.json:20]: Should not be needed AFAIK

**Comment by travier** [comment]
```
builddir/files/share/appdata/org.kde.kmplot.appdata.xml: FAILED:
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
```

Classic missing OARS data issue :)

**Comment by Mailaender** [comment]
bot, build org.kde.kmplot

**Comment by cho2** [comment]
Cuplikan layar dari 2022-05-17 13-57-14

**Comment by travier** [comment]
Manifest LGTM

**Comment by tsdgeos** [comment]
Yeah let's go with it

**Pull request closed without merging**